templates/bundles/KnpPaginatorBundle/Pagination/bootstrap_v5_pagination.html.twig

```diff
@@ -1,8 +1,8 @@
 {% if pageCount > 1 %}
   <nav>
-    {% set classAlign = (align is not defined) ? '' : align=='center' ? ' justify-content-center' : (align=='right' ? ' justify-content-end' : '') %}
-    {% set classSize = (size is not defined) ? '' : size=='large' ? ' pagination-lg' : (size=='small' ? ' pagination-sm' : '') %}
-    <ul class="pagination{{ classAlign }}{{ classSize }}">
+    {% set class_align = (align is not defined) ? '' : align == 'center' ? ' justify-content-center' : (align == 'right' ? ' justify-content-end' : '') %}
```

Comment:
Die mag je toch niet zomaar aanpassen? Is dat geen variabele die in andere templates van de bundle kan gebruikt worden?

Reply:
De variable die hier geset worden zijn aan de hand van align en size, class_align en class_size worden hier in de locale scope gebruikt